mllib/src/main/scala/org/apache/spark/ml/feature/QuantileDiscretizer.scala

```diff
@@ -70,17 +70,16 @@ private[feature] trait QuantileDiscretizerBase extends Params
    * invalid values), error (throw an error), or keep (keep invalid values in a special additional
    * bucket).
    * Default: "error"
+   * TODO: Reuse handleInvalid in HasHandleInvalid.
```

Comment:
Is this just a note to add "HasHandleInvalid" as a shared param? I was a bit confused by it at first. Maybe we can reference a jira number for it?